src/components/DrumPad.js

```diff
@@ -10,11 +10,18 @@ export default function DrumPad({value, handleClick}){
 
 const Pad = styled.button`
 
-  background-color: blue;
+  background-color: #4895EF;
```

Comment:
With Tanja I have seen folegendes and would also recommend it to you.
In the GlobalStyles.js you can use :root{
--color-dark-black: #4895EF} to define your colors in a central place and then call them with var(--color...) in the styled-components.
Advantage you can then globally change the colors. 
I would also add the boxshadow.

Reply:
Nice idea. I'll add it to the fourth user story when the settings for the colors come up.